src/gqlTypes/globalTypes.ts

```diff
@@ -327,6 +332,16 @@ export enum CountryCode {
   ZW = "ZW",
 }
 
+/**
+ * An enumeration.
```

Comment:
That's an example of the unnecessary comment. It provides no additional context.